Document/0x05g-Testing-Network-Communication.md

```diff
@@ -169,6 +169,8 @@ If a certificate pinning validation check has failed, the following event will b
 I/X509Util: Failed to validate the certificate chain, error: Pin verification failed
 ```
 
+As defined in the previous example, a \<pin-set\> can have a defined expiration date. When the expiration date is reached, the network communication will continue to work, but the Certificate Pinning will be disabled for the affected domains. Be aware of this behavior.
```

Comment:
Please put this sentence into the network security config test case below. So it's only described one time.

Reply:
Fixed.